storage/engine/mvcc.go

```diff
@@ -47,11 +47,7 @@ const (
 //  - Key count (count of all keys, including keys with deleted tombstones)
 //  - Value count (all versions, including deleted tombstones)
 //  - Intents (provisional values written during txns)
-type MVCCStats struct {
-	LiveBytes, KeyBytes, ValBytes, IntentBytes int64
-	LiveCount, KeyCount, ValCount, IntentCount int64
-	IntentAge, GCBytesAge, LastUpdateNanos     int64
-}
+type MVCCStats proto.MVCCStats
```

Comment:
I think you should actually go ahead and update the usage of MVCCStats in storage/engine and storage to use proto directly. Just a simple search and replace.